theano/tensor/tests/test_basic.py

```diff
@@ -6863,30 +6877,31 @@ def test_infer_shape(self):
         #                        [Reshape(ndim)(adtens4, aivec)],
         #                        [adtens4_val, [1, 3, 10, 4]], Reshape)
 
-        # Tile
-        advec = dvector()
-        advec_val = rand(5)
-        aivec_val = [3]
-        ndim = 1
-        self._compile_and_check([advec],
-                                [tile(advec, aivec_val, ndim)],
-                                [advec_val], Tile)
-
-        admat = dmatrix()
-        admat_val = rand(2, 4)
-        aivec_val = [2, 3]
-        ndim = None
-        self._compile_and_check([admat],
-                                [tile(admat, aivec_val)],
-                                [admat_val], Tile)
-
-        adtens4 = dtensor4()
-        adtens4_val = rand(2, 4, 3, 5)
-        aivec_val = [2, 3, 1, 4]
-        ndim = 4
-        self._compile_and_check([adtens4],
-                                [tile(adtens4, aivec_val, ndim)],
-                                [adtens4_val], Tile)
+        # Tile op is depricated
+        
+        # advec = dvector()
+        # advec_val = rand(5)
+        # aivec_val = [3]
+        # ndim = 1
+        # self._compile_and_check([advec],
+        #                         [tile(advec, aivec_val, ndim)],
+        #                         [advec_val], Tile)
+
+        # admat = dmatrix()
+        # admat_val = rand(2, 4)
+        # aivec_val = [2, 3]
+        # ndim = None
+        # self._compile_and_check([admat],
+        #                         [tile(admat, aivec_val)],
+        #                         [admat_val], Tile)
+
+        # adtens4 = dtensor4()
+        # adtens4_val = rand(2, 4, 3, 5)
+        # aivec_val = [2, 3, 1, 4]
+        # ndim = 4
+        # self._compile_and_check([adtens4],
+        #                         [tile(adtens4, aivec_val, ndim)],
+        #                         [adtens4_val], Tile)
```

Comment:
The op is deprecated but some people might still be using it. This means that, as much as possible, instead of deleting these tests, they should be adapted to directly use the op Tile.